docs/machine-learning/how-to-guides/getting-started-dataframe.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting started with DataFrames
+description: Learn how to use DataFrame to manipulate and prepare data.
+author: beccam
+ms.author: beccam
+ms.date: 1/13/2023
+ms.custom: mvc, how-to, title-hack-0625
+ms.topic: how-to
+---
+
+# What is the DataFrame?
+
+Learn how to get started with DataFrames. [DataFrames](https://learn.microsoft.com/dotnet/api/microsoft.data.analysis.dataframe) are a two-dimensional data structure for storing and manipulating data. DataFrames help with preparation of data for a machine learning model. DataFrames can also be used for data manipulation unrelated to machine learning. 
+
+# Install Microsoft.Data.Analysis
+
+In most cases, accessing DataFrame is as simple as referencing the Microsoft.Data.Analysis NuGet package.
+
+```dotnetcli
+dotnet add package Microsoft.Data.Analysis
+```
+
+# Load data
+
+DataFrames make it easy to load tabular data. Create a comma-separated file called housing-prices.csv with the following data. 
+
+```text
+Id,Size,HistoricalPrice,CurrentPrice
+1,600f,100000,170000
+2,1000f,200000,225000
+3,1000f,126000,195000
+4,850f,150000,205000
+5,900f,155000,210000
+6,550f,99000,180000
+```
+
+Start by loading the data into a DataFrame. 
+
+```csharp
+using System.IO;
+using System.Linq;
+using Microsoft.Data.Analysis;
+
+// Define data path
+var dataPath = Path.GetFullPath(@"housing-prices.csv");
+
+// Load the data into the data frame
+var dataFrame = DataFrame.LoadCsv(dataPath);
+```
+
+# Inspect Data (Info/Describe)
+
+DataFrames store data as a collection of columns. This makes it easy to interact with the data. 
+
+To get a preview of the column datatypes, run `Info()`.
```

Comment:
Should this link to the doc on info?

Reply:
Yes please. Whenever we reference APIs, let's try to link to the API docs.